### ISSUE TITLE
Bug: Node.js YouTube Channel video uploads are disappearing

### ISSUE THREAD

**Issue by obensource** [nodejs/admin#487]
## Description
All new Node.js WG meeting videos that are streamed live to our [YouTube channel](https://www.youtube.com/channel/UCQPYJluYC_sn_Qz_XE-YbTQ/videos) via Zoom (eg. from CommComm's Zoom account) are disappearing.

## Behavior
The last video that has persisted to YouTube was from the Build WG meeting, but today's i18n and Tooling WG videos weren’t uploaded (it looks like).

👉  Additionally, what's strange is that the title of the latest video changes to whichever WG meeting attempted to upload last (and just keeps changing the title of [the Build WG video](https://www.youtube.com/watch?v=kE1mY_k5oW4)).

## Help
This seems like it could be a bug in YouTube, and/or the Zoom/YouTube integration. Is there anyone from Google, YouTube, or Zoom here who's seen this bug before?

**Comment by sam-github** [comment]
@mhdawson is aware of this, and has reached out to the Foundation to see if they can help (we are paid Zoom customers so can hopefully get help).

So, definitely a problem, and its being worked on!

**Comment by brianwarner** [comment]
Yes, something strange is going on. @mhdawson and I were working on it yesterday, and we filed a support ticket. We'll be in touch when we hear from them. In the meantime, I'd recommend doing a cloud recording of each call, downloading it, and then posting it directly to the channel.

**Comment by obensource** [comment]
Looks like this issue was resolved a couple days ago and videos are being uploaded properly again.

Maybe @mhdawson might know if the videos between March 26th and April 6th (I think that timeline is correct) were clobbered and no longer exist, or if they're recoverable (no pressure either way, just curious).

I went ahead and updated the [Release working group meeting video](https://www.youtube.com/watch?v=kE1mY_k5oW4) to have the correct title.

Given that videos are persisting again, I'm going to consider this issue closed. Thanks all! 🙏

**Comment by obensource** [comment]
Hi all! 👋

Reopening this issue since we're still experiencing streaming issues. I tried starting the live stream twice today when kicking off an i18n meeting, and both attempts failed.

@brianwarner & @mhdawson: any luck with the support ticket?

I also noticed that the description for a lot of our meetings point to an old OpenJS CPC bootstrap meeting, and that the title of the Release working group video which I initially pointed out still keeps changing to which ever working group last had a failed attempt to stream/upload a video.

i18n WG was the last to attempt to stream/upload, so we see this bug reflected here:

<img width="726" alt="Screen Shot 2020-04-10 at 3 21 10 PM" src="https://user-images.githubusercontent.com/1910457/79026962-4d676800-7b3f-11ea-9da2-ab5451d2981a.png">

I changed the video title [here](https://www.youtube.com/watch?v=kE1mY_k5oW4&feature=youtu.be) accordingly after taking this screenshot, but it will likely be clobbered next time someone kicks off a stream, just fyi.

As @brianwarner said above, let's encourage folks to upload via cloud or manually for now until this is resolved. Thanks a million y'all! 🙏

**Comment by mhdawson** [comment]
I think I fixed it by making the "stuck" video private. After that the build WG meeting streamed ok.  We should leave this open until a few more people confirm its working again.
